internal/server/handlers/models/model.go

```diff
@@ -29,4 +29,10 @@ type (
 		CorrelationID string `json:"correlation_id"`
 		ShortURL      string `json:"short_url"`
 	}
+
+	// RespStats model for response Stats request.
+	RespStats struct {
```

Comment:
```suggestion
	ResponseStats struct {
```
не вижу смысла в экономии 5 букв. Или даже ResponseStatus

Reply:
fixed